.github/workflows/merge-when-ready.yml

```
@@ -69,6 +69,217 @@ jobs:
+              uses: actions/download-artifact@v7
+              with:
+                  name: runtimes
+                  path: target/release/wbuild/${{ matrix.chains.runtime }}-runtime/
+            # Binaries needed for testPalletVersions test
+            - name: "Download binaries"
+              uses: actions/download-artifact@v7
+              with:
+                  name: binaries
+                  path: target/release
+            - name: Make binaries executable
+              uses: ./.github/workflow-templates/make-binaries-executable
```

actionlint (1.7.10)

description is required in metadata of "Make binaries executable" action at "/home/jailuser/git/.github/workflow-templates/make-binaries-executable/action.yml"

(action)